codespell_lib/data/dictionary.txt

```diff
@@ -10500,41 +10590,60 @@ transmiting->transmitting
 transmition->transmission
 transmitsion->transmission
 transmittion->transmission
+transmitts->transmits
 transorm->transform
 transormed->transformed
 transorming->transforming
 transorms->transforms
 transparancy->transparency
 transparant->transparent
 transparantly->transparently
+transparecy->transparency
+transpeorted->transported
 transporation->transportation
+transportatin->transportation
 transtion->transition
 transtions->transitions
 transtition->transition
 transtitions->transitions
 transtorm->transform
 transtormed->transformed
+transverse->traverse
```

Comment:
"transverse" is a word (e.g., "transverse axis")

Reply:
removed